src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+  let vocationCost = 0;
+
+  if (days >= 7) {
+    vocationCost += days * cost - bigDiscount;
+  } else if (days >= 3) {
+    vocationCost += days * cost - smallDiscount;
+  } else {
+    vocationCost += days * cost;
+  }
```

Comment:
Prefer using 'const' over 'let' for 'vocationCost' if you're not planning to reassign it. In this case, you can calculate the cost directly in the return statement, which makes the 'let' unnecessary.